libyul/optimiser/Suite.cpp

```diff
@@ -445,7 +483,14 @@ void OptimiserSuite::runSequence(std::vector<string> const& _steps, Block& _ast)
 	{
 		if (m_debug == Debug::PrintStep)
 			cout << "Running " << step << endl;
+#ifdef OUTPUT_PERFORMANCE_METRICS
+		chrono::steady_clock::time_point startTime = chrono::steady_clock::now();
+#endif
 		allSteps().at(step)->run(m_context, _ast);
+#ifdef OUTPUT_PERFORMANCE_METRICS
+		chrono::steady_clock::time_point endTime = chrono::steady_clock::now();
+		m_durationPerStepInMicroseconds[step] += std::chrono::duration_cast<std::chrono::microseconds>(endTime - startTime).count();
```

Comment:
We use the map because it's convenient to access the specific step, since each step can be run multiple times, right? So this is essentially the sum of execution times for each step. Would it make sense to differentiate even more, i.e. provide execution times per run of same step?